core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -97,6 +97,8 @@
  * @property boolean $hasMooTools
  * @property string  $template
  * @property string  $templateGroup
+ * @property boolean $enforceTwoFactor
```

Comment:
We also have to add the method stubs for `findOneByEnforceTwoFactor()` etc. below.

Reply:
Fixed in 5f7de11.